spa/src/components/pageEditor/elementEditors/amount/AmountItem.styled.ts

```diff
@@ -0,0 +1,36 @@
+import { ButtonBase } from '@material-ui/core';
+import { Close } from '@material-ui/icons';
+import styled, { css } from 'styled-components';
+
+export const Root = styled.div<{ $isDefault: boolean }>`
+  align-items: center;
+  border-radius: ${({ theme }) => theme.muiBorderRadius.lg};
+  box-shadow: 0 0.3px 0.5px rgba(0, 0, 0, 0.02), 0px 2px 4px rgba(0, 0, 0, 0.08);
+  color: ${({ theme }) => theme.colors.muiGrey[900]};
+  display: inline-flex;
+  font-size: ${({ theme }) => theme.fontSizesUpdated.md};
+  height: 35px;
+  min-width: 100px;
+  padding: 10px;
+
+  ${(props) => {
+    if (props.$isDefault) {
+      return css`
+        background: ${({ theme }) => theme.colors.muiLightBlue[800]};
+        box-shadow: none;
+        color: ${({ theme }) => theme.colors.white};
+      `;
+    }
+  }}
```

Comment:
Can simplify this style: 

```suggestion
  ${(props) =>
    props.$isDefault &&
      `
        background: ${({ theme }) => theme.colors.muiLightBlue[800]};
        box-shadow: none;
        color: ${({ theme }) => theme.colors.white};
      `
  }
```

Reply:
If I drop the `css` I get this error: `Binding element 'theme' implicitly has an 'any' type.` Do you know if there's a type annotation I can put on the `theme` args that will make TypeScript happy?

Reply:
I've updated my suggestion in a way that might prevent this error.

Reply:
:+1: aa523ee282cb2f3b15d95b1881eaf7bf5cb7569a